apps/src/templates/projects/PersonalProjectsTable.jsx

```diff
@@ -89,7 +89,32 @@ const isPublishedFormatter = function (isPublished) {
 const actionsFormatter = function (actions, {rowData}) {
   return (
     <QuickActionsCell>
-      <ProjectActionBox isPublished={rowData.isPublished}/>
+      <QuickAction
+        text={i18n.rename()}
+        action={()=>{}}
```

Comment:
This table isn't done and is currently only in storybook, adding placeholder actions for now.

Reply:
this is fine for a placeholder. once you have actual functions in here, you'll want to define them outside the render method so that we don't re-bind to this on every render